lib/webpacker.rb

```diff
@@ -2,14 +2,15 @@ module Webpacker
   extend self
 
   def bootstrap
-    Webpacker::Env.load
-    Webpacker::Configuration.load
-    Webpacker::Manifest.load
+    Webpacker::Env.load_instance
+    Webpacker::Configuration.load_instance
+    Webpacker::DevServer.load_instance
+    Webpacker::Manifest.load_instance
   end
 
   def compile
     Webpacker::Compiler.compile
-    Webpacker::Manifest.load
+    Webpacker::Manifest.load_instance
```

Comment:
Concur with @gauravtiwari on this being unrelated to this PR. I also don't think it's an improvement but we could discuss that in another ticket.